NetKAN/RealPlume.netkan

```diff
@@ -1,33 +1,27 @@
 {
-    "spec_version"   : "v1.4",
-    "$kref"          : "#/ckan/github/KSP-RO/RealismOverhaul",
-    "$vref"          : "#/ckan/ksp-avc",
-    "name"           : "Real Plume",
-    "identifier"     : "RealPlume",
-    "abstract"       : "Option install for Realism Overhaul, adds realistic plume expansion to engines.  See thread for currently configured engines.",
-    "license"        : "CC-BY-SA",
-    "release_status" : "stable",
+    "spec_version"      : "v1.4",
+    "$kref"             : "#/ckan/github/KSP-RO/RealPlume",
+    "ksp_version"       : "1.0"
+    "name"              : "Real Plume",
+    "identifier"        : "RealPlume",
+    "abstract"          : "A set of effects and basic configs that can be attached to any engine as appropriate.  Requires secondary configs to function correctly.",
+    "license"           : "CC-BY-NC-SA",
+    "release_status"    : "WIP",
```

Comment:
`release_status` is an enumeration so it can only take one of [three fixed values](https://github.com/KSP-CKAN/CKAN/blob/master/CKAN.schema#L77). `development` is probably the appropriate one here.